Description: Build with debug symbols
 In order to create -dbg packages on Debian we need to build with the debug
 flag.
Forwarded: not-needed
Author: Martin Erik Werner <martinerikwerner@gmail.com>
Last-Update: 2011-12-13

Index: redeclipse-1.3.1/src/Makefile
===================================================================
--- redeclipse-1.3.1.orig/src/Makefile	2012-08-31 07:59:06.000000000 +0200
+++ redeclipse-1.3.1/src/Makefile	2012-09-02 12:31:30.062739514 +0200
@@ -3,7 +3,7 @@
 APPSERVER=reserver
 
 #CXXFLAGS= -ggdb3
-CXXFLAGS= -O3 -fomit-frame-pointer
+CXXFLAGS= -O3 -fomit-frame-pointer -g
 override CXXFLAGS+= -Wall -fsigned-char -fno-exceptions -fno-rtti
 
 PLATFORM= $(shell uname -s)
